action.yml

```diff
@@ -24,11 +24,11 @@ branding:
 runs:
   using: composite
   steps:
-    # Set up a non-EOL, cibuildwheel & pipx supported Python version
+    # Set up the version of Python that supports pyodide
     - uses: actions/setup-python@v5
       id: python
       with:
-        python-version: "3.8 - 3.12"
+        python-version: "3.12"
```

Comment:
Unrelated to the PR but I was thinking that maybe, cibuildwheel could follow [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html) ?

Reply:
That's now superseded by [SPEC 0](https://scientific-python.org/specs/spec-0000/). Is there a reason to? I personally prefer official EoL, though for cibuildwheel, it really doesn't matter as much, since CI or developers should have a recent Python; it's not user-facing. Any particular reason for being interested in following NEP 29/SPEC 0, 3.10+?

Reply:
Thanks for the link, I somehow missed the prominent note in NEP 29...

> Is there a reason to?

Nothing else than it allows to get newer CPython features faster and, given the specific nature of cibuildwheel which you mentioned, I thought it might be interesting to follow the schedule of SPEC 0.

python 3.10 might not change much (mostly some typing,  `functools.cache`). With Python 3.11+ at the end of the year we could drop all things in `_compat` and remove those conditionnal dependencies.

I might open a discussion about this at some point.